cli/cleanup/pods.go

```diff
@@ -0,0 +1,63 @@
+package cleanup
+
+import (
+	"log"
+
+	"github.com/deliveroo/paddle/cli/pipeline"
+	"github.com/spf13/cobra"
+	k8errors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var clientset kubernetes.Interface
+var logFatalf = log.Fatalf
+
+var podsCmd = &cobra.Command{
+	Use:   "pods",
+	Short: "Clean up all finished pods",
+	Long: `Fetch all pods and delete the ones in complete state
+
+Example:
+
+$ paddle cleanup pods
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		runPodsCleanup()
+	},
+}
+
+func init() {
+	config, err := pipeline.GetKubernetesConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+	clientset, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+}
+
+func runPodsCleanup() {
+	pods := clientset.CoreV1().Pods("")
+	podList, err := pods.List(metav1.ListOptions{})
+	if err != nil {
+		logFatalf("[paddle] error fetching list of pods: %s", err.Error())
+	}
+
+	for _, pod := range podList.Items {
+		switch string(pod.Status.Phase) {
+		case "Succeeded", "Failed", "Unknown":
```

Comment:
I've never seen `Unknown` state before. Also for `Failed` - i'd suggest you only delete those if they've been around for more than 1 hour - to allow people to inspect the logs.